cmd/adapter/adapter.go

```diff
@@ -144,6 +161,12 @@ func (cmd *PrometheusAdapter) addFlags() {
 		"interval at which to re-list the set of all available metrics from Prometheus")
 	cmd.Flags().DurationVar(&cmd.MetricsMaxAge, "metrics-max-age", cmd.MetricsMaxAge, ""+
 		"period for which to query the set of available metrics from Prometheus")
+	cmd.Flags().StringVar(&cmd.secureMetricsPort, "secure-metrics-port", cmd.secureMetricsPort,
```

Comment:
Users shouldn't be only able to configure secure ports, they should be able to choose whatever fits their needs. As such I wouldn't mention the `secure` part in the option name.